src/app/utils/index.ts

```diff
@@ -78,3 +78,15 @@ export async function getAlbyWalletOptions() {
     console.error(e);
   }
 }
+
+export function extractLightningTagData(url: string) {
+  const reqExp = /lightning=([^&|\b]+)/i;
```

Comment:
We should add a comment here why we search for that (basically for LNBits links iirc)